appium/options/android/common/avd/avd_options.py

```diff
@@ -0,0 +1,217 @@
+    """
+    Gets and Sets emulator command line arguments.
+
+    Usage
+    -----
+    - Get
+        - `self.avd_args`
+    - Set
+        - `self.avd_args` = `value`
+    
+    Parameters
+    ----------
+    `value`: `str`
+
+    Returns
+    -------
+    - Get
+        - `Optional[str]`
+    - Set
+        - `None`
+    """
+
+
+class AvdEnvOption(SupportsCapabilities):
+    AVD_ENV = "avdEnv"
+    avd_env = OptionsDescriptor[Optional[Dict[str, str]], str](AVD_ENV)
+    """
+    Gets and Sets the mapping of emulator environment variables.
+
+    Usage
+    -----
+    - Get
+        - `self.avd_env`
+    - Set
+        - `self.avd_env` = `value`
+
+    Parameters
+    ----------
+    `value`: `str`
+
+    Returns
+    -------
+    - Get
+        - `Optional[Dict[str, str]]`
+    - Set
+        - `None`
+    """
+
+
+class AvdLaunchTimeoutOption(SupportsCapabilities):
+    AVD_LAUNCH_TIMEOUT = "avdLaunchTimeout"
+    avd_launch_timeout = OptionsDescriptor[Optional[timedelta], Union[timedelta, int]](AVD_LAUNCH_TIMEOUT)
+    """
+    Timeout to wait until Android Emulator is started.
+    Maximum timeout to wait until Android Emulator is started.
+    60000 ms by default.
+
+    Usage
+    -----
+    - Get
+        - `self.avd_launch_timeout`
+    - Set
+        - `self.avd_launch_timeout` = `value`
+    
+    Parameters
+    ----------
+    `value`: `Union[timedelta, int]`
+
+    Returns
+    -------
+    - Get
+        - `Optional[timedelta]`
+    - Set
+        - `None`
+    """
+
+
+class AvdOption(SupportsCapabilities):
+    AVD = "avd"
+    avd = OptionsDescriptor[Optional[str], str](AVD)
+    """
+    The name of Android emulator to run the test on.
+    Names of currently installed emulators could be listed using
+    avdmanager list avd command. If the emulator with the given name
+    is not running then it is going to be started before a test.
+
+    Usage
+    -----
+    - Get
+        - `self.avd`
+    - Set
+        - `self.avd` = `value`
+    
+    Parameters
+    ----------
+    `value`: `str`
+
+    Returns
+    -------
+    - Get
+        - `Optional[str]`
+    - Set
+        - `None`
+    """
+
+
+class AvdReadyTimeoutOption(SupportsCapabilities):
+    AVD_READY_TIMEOUT = "avdReadyTimeout"
+    avd_ready_timeout = OptionsDescriptor[Optional[timedelta], Union[timedelta, int]](
+        AVD_READY_TIMEOUT, transform_duration_get, transform_duration_set
+    )
+    """
+    Maximum timeout to wait until Android Emulator is fully booted and is ready for usage.
+    60000 ms by default
+
+    Usage
+    -----
+    - Get
+        - `self.avd_ready_timeout`
+    - Set
+        - `self.avd_ready_timeout` = `value`
+    
+    Parameters
+    ----------
+    `value`: `Union[timedelta, int]`
+
+    Returns
+    -------
+    - Get
+        - `Optional[timedelta]`
+    - Set
+        - `None`
+    """
+
+
+class GpsEnabledOption(SupportsCapabilities):
+    GPS_ENABLED = "gpsEnabled"
```

Comment:
double quotes here instead of single

Reply:
same below

Reply:
> double quotes here instead of single

I had used single quotes. but after formatting it has changed to double quotes..

Reply:
hm, this is some strange black config

@KazuCocoa does it do same changes for you by replacing single quotes with double ones?

Reply:
@mykola-mokhnach I saw the source code after formatting and I was surprised to see all single quotes converted to double quotes.. I thought it is taking as per config.. so I did not change it in source code..

Reply:
Hm, I thought we used single quote. Actually selenium Python binging required double quote though.